Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/threshold/ThresholdedRandomCutForest.java

```diff
@@ -488,4 +805,358 @@ public void setLastExpectedPoint(double[] point) {
         return array == null ? null : Arrays.copyOf(array, array.length);
     }
 
+    /**
+     * maps a value shifted to the current mean or to a relative space
+     * 
+     * @param value     input value of dimension
+     * @param normalize flag that determines relative or shift
+     * @param deviation statistic
+     * @return the normalized value
+     */
+    double normalize(double value, boolean normalize, Deviation deviation) {
```

Comment:
It seems weird to pass a boolean to this method instead of making that decision one level up before this method is called.